website/content/docs/oss/configuration/worker/pki-worker.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: PKI Worker Configuration
+description: |-
+  PKI worker-specific parameters.
+---
+
+
+## PKI Worker Configuration
+PKI Workers authenticate to Boundary using a new certificate-based method, allowing for worker deployment
```

Comment:
```suggestion
PKI Workers authenticate to Boundary using a certificate-based method, allowing for worker deployment
```
Unless we remember to remove "new" later it will just be out of date...I don't think we really need it here at all